doc/source/configuration/cephadm.rst

```diff
@@ -308,6 +308,128 @@ should be used in the Kolla Manila configuration e.g.:
 
   manila_cephfs_filesystem_name: manila-cephfs
 
+RADOS Gateways
+--------------
+
+RGWs are defined with the following:
+
+.. code:: yaml
+
+  cephadm_radosgw_services:
+    - id: myrgw
+      count_per_host: 1
+      spec:
+        rgw_frontend_port: 80
```

Comment:
`80` is rather unusual choice - does not correspond with an example from hyper-converged section too. Normally, web services like RGW have ports >1024 allocated.

Reply:
I've changed to suggest port 8100 by default, as afaik this doesn't clash with any other services we deploy.